programs/steward/src/state/steward_state.rs

```diff
@@ -967,15 +967,6 @@ impl StewardState {
                 RebalanceType::None
             };
 
-            msg!("Reserve lamports after adjustment: {}", reserve_lamports);
```

Comment:
Too much CU?

Reply:
was causing local tests to fail since it was calling msg! outside of context, + think we're okay to remove these now since we have the emits

Reply:
10-4